packages/peregrine/lib/talons/Image/useResourceImage.js

```diff
@@ -21,16 +22,17 @@ export const useResourceImage = props => {
         resource,
         type,
         width,
-        widths
+        widths,
+        ratio
     } = props;
 
     const src = useMemo(() => {
         return generateUrl(resource, type)(width, height);
```

Comment:
Ratio should also be accounted for here. Some browsers load src and srcset side by side, and could cause the container to be the wrong size (without the right ratio).

Reply:
Because, after this commit 5536d6156d5d4546aa56a7914e76a764e10464b3, the height sent to `ResourceImage` and then to `useResourceImage` is checked against the ratio. I don't think we will need this any more.